### DISCUSSION TITLE
Jellyfin

### DISCUSSION THREAD

**Discussion by sn0opy** [selfhst/icons#6] [New Icon Request]
https://jellyfin.org/

General branding info: https://jellyfin.org/docs/general/contributing/branding/
Link to SVGs: https://github.com/jellyfin/jellyfin-ux/tree/master/branding/SVG

**Comment by shollyethan** [comment]
Jellyfin was already in the collection, but wasn't in the data file that populates the site. Thanks for letting me know - I also added a light version while I was checking.
